solidity/solc-update-0.8.15-to-0.8.25.md

```diff
@@ -0,0 +1,98 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Solidity Version Upgrade Proposal](#solidity-version-upgrade-proposal)
+  - [Benefits to the OP Stack](#benefits-to-the-op-stack)
+      - [DevX](#devx)
+      - [Gas Optimization](#gas-optimization)
+      - [Features](#features)
+  - [Notable Features](#notable-features)
+  - [Notable Bug Fixes](#notable-bug-fixes)
+  - [Changelog Links](#changelog-links)
+  - [Action Items](#action-items)
+  - [Additional Notes](#additional-notes)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+|                     |                |
+| ------------------- | -------------- |
+| Author              | Michael Amadi  |
+| Created at          | 2024-10-02     |
+| Needs Approval From | Kelvin Fichter |
+| Other Reviewers     | Matt Solomon   |
+| Status              | In Review      |
+
+# Solidity Version Upgrade Proposal
+
+- **Current Version:** 0.8.15, 0.8.19, 0.8.25.
+- **Proposed Version:** 0.8.25.
+
+## Benefits to the OP Stack
+
+Explain specific feature(s) in the new version that would benefit the OP Stack codebase, with a
+clear explanation of their potential impact and usefulness.
+
+#### DevX
+
+- Allow named parameters in mapping types (>=0.8.18): Limits error when reading mappings of high dimensions when called using named parameters.
```

Comment:
This is saying if I have:
```solidity
mapping (address from => mapping (address to => uint256)) public allowance;
```

I can call it like this?
```solidity
allowance({from: me, to: you})
```

That's cool, I didn't realized the named param syntax worked for mappings

Reply:
Wait, actually this isn't possible 🤦‍♂️, will remove it